cmd/root.go

```diff
@@ -151,21 +151,32 @@ func beginDetectNewVersion() chan interface{} {
 			return
 		}
 
+		blockedVersions := common.Blobfuse2ListContainerURL + "/blockedversions/" + common.Blobfuse2Version
+		isBlocked := checkVersionExists(blockedVersions)
+
+		if isBlocked {
+			blockedPage := common.BlobFuse2BlockingURL + "#" + strings.ReplaceAll(strings.ReplaceAll(common.Blobfuse2Version, ".", ""), "~", "")
+			fmt.Fprintf(stderr, "Visit %s to see the list of known issues blocking your current version [%s]\n", blockedPage, common.Blobfuse2Version)
+			log.Warn("Visit %s to see the list of known issues blocking your current version [%s]\n", blockedPage, common.Blobfuse2Version)
+			os.Exit(1)
+		}
+
+		warningsUrl := common.Blobfuse2ListContainerURL + "/securitywarnings/" + common.Blobfuse2Version
+		hasWarnings := checkVersionExists(warningsUrl)
+
+		if hasWarnings {
+			warningsPage := common.BlobFuse2WarningsURL + "#" + strings.ReplaceAll(strings.ReplaceAll(common.Blobfuse2Version, ".", ""), "~", "")
+			fmt.Fprintf(stderr, "Visit %s to see the list of known issues associated with your current version [%s]\n", warningsPage, common.Blobfuse2Version)
+			log.Warn("Visit %s to see the list of known issues associated with your current version [%s]\n", warningsPage, common.Blobfuse2Version)
+		}
+
```

Comment:
It's better to check if the version has a security warning first then check if the version is blocked.

Reply:
updated. Kindly review